core/templates/dev/head/pages/exploration_editor/issues_tab/IssuesTab.js

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controllers for the exploration issues tab in the exploration
+ * editor.
+ */
+
+oppia.controller('IssuesTab', ['$scope', function($scope) {
+  $scope.resolvedIssueCount = 13;
```

Comment:
Could these magic numbers instead be represented as Angular constants? I recommend adding some brief documentation to the constant declarations, too, to provide context on why these values are used (consider linking to your design doc if that's where the context exists).